### PR TITLE
Add more meta data for samples

### PR DESCRIPTION
**Pull request by ChristineP2**
Added 3 meta data keys to the stubs (mv_title, mv_description, mv_classes). These additions allow for detailed information to be provided on a per example basis. I did test in my fork of mountain_view_demo. This is another change referenced in #70

Added comments to #70 with screen caps

Second attempt

### CODE REVIEW

File: app/assets/stylesheets/mountain_view/styleguide.css [hunk 1]
```diff
@@ -18,6 +18,12 @@ body{
   background: #777;
   padding: 20px;
 }
+
+.mv--black-background{
```
Comment (kitop) [inline app/assets/stylesheets/mountain_view/styleguide.css]: Do we need this here?
Seems not to be used but by the examples. If so, that's something the user should provide as part of their customization, as we try to keep the styles to only the ones used by the default views.

File: app/views/mountain_view/styleguide/show.html.erb [hunk 2]
```diff
@@ -12,15 +12,24 @@
     <% if @component.component_stubs? %>
       <% @component.component_stubs.each_with_index do |component_properties, index| %>
           <div class="mv-component">
-            <div class="mv-component__item">
+            <% if @component.example_classes(component_properties).blank? %>
```
Comment (kitop) [inline app/views/mountain_view/styleguide/show.html.erb]: We can get rid of this if and always interpolate the extra classes. If they are blank, it just won't add anything but a blank space which is still valid HTML.

File: app/assets/stylesheets/mountain_view/styleguide.css (hunk 1 above)
Comment (ChristineP2) [reply app/assets/stylesheets/mountain_view/styleguide.css]: Good catch, I'd put it in for the screen shot and forgot to take it out... oops!

**Comment by kitop** [comment]
Sounds good!! Again, thanks a lot for contributing!

**Comment by ChristineP2** [comment]
I think I will also need to update the partial `_example.html.erb`

**Comment by kitop** [comment]
Oh, good catch! it's not 100% needed as those keys are optional and will be in the documentation, but would be nice to have them more exposed, yes.

**Pull request closed without merging**